_opensearch/query-dsl/text-analyzers.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: Text analyzers
+parent: Query DSL
+nav_order: 41
+---
+
+
+# Text analyzers
```

Comment:
Do we just want to call this section "Analyzers" and include a sentence that mentions that analysis is only used on `text` fields?